libs/designsystem/src/lib/components/action-sheet/action-sheet.component.scss

```
@@ -0,0 +1,23 @@
+@import '../../scss/utils';
+
+:host {
+  position: relative;
+  display: inline-block;
+  outline: none;
+}
+
+:host(.is-open) {
+  kirby-action-sheet-popout {
+    display: block;
+  }
+  button {
+    box-shadow: get-elevation(8);
+  }
+}
+
+kirby-action-sheet-popout {
+  display: none;
+  position: absolute;
+  right: 0;
+  margin-top: size('xs');
```

I believe the issues states that spacing should be `s`/ `16px`? 🤔 
```suggestion
  margin-top: size('s');
```

---

Yup, but adding the `4px` button margin `size('xs')` fits the bill :)